x-pack/plugins/observability/server/ui_settings.ts

```diff
@@ -187,4 +188,15 @@ export const uiSettings: Record<string, UiSettingsParams<boolean | number | stri
     requiresPageReload: false,
     type: 'boolean',
   },
+  [serviceGroupMaxNumberOfServices]: {
```

Comment:
There was a discussion to prefix all our settings with `apm`. Could you add this here please?